column_descriptions/posebusters_checks.tsv

```diff
@@ -1,126 +1,126 @@
 Name	Type	Description
```

Comment:
As this is a TSV file, `tablegen.py` expects the columns to be separated by tab characters. This way also a new column is distinguished from whitespace between words in the descriptions.

In consequence, currently the descriptions of these annotations end up in the type column

Reply:
Makes sense.